### ISSUE TITLE
Venue: Custom Max Papers shouldn't allow more than one edge with the same head and tail

### ISSUE THREAD

**Issue by melisabok** [openreview/openreview-py#1976]
This is to avoid PCs creating duplicated quotas by mistake.

I remember that using `maxReplies` = 1 validates this, @celestemartinez can you confirm that?

**Comment by celestemartinez** [comment]
Yes, `maxReplies=1` should be enought to make sure PCs can't post multiple edges and they edit the existing ones.
